controllers/vrg_volrep.go

```diff
@@ -508,13 +507,13 @@ func (v *VRGInstance) PVUploadToObjectStores(pvc *corev1.PersistentVolumeClaim,
 			rmnutil.ReportIfNotPresent(v.reconciler.eventRecorder, v.instance, corev1.EventTypeWarning,
 				rmnutil.EventReasonPVUploadFailed, err.Error())
 
-			return succeededProfiles, err
+			returnErr = err
```

Comment:
If we fail to upload a PV to the store, we should stop attempting to load to the store as well. IOW, instead of failing for 'n' PVs with the same error, we should break out of the current loop.

Currently, even on error line 513 below would be executed, and the profile considered a successful upload.